daemon/remote/cmd/cmuxd-remote/agent_launch.go

```
@@ -146,6 +148,79 @@ func runOMORelay(socketPath string, args []string, refreshAddr func() string) in
+			os.Setenv("CMUX_SURFACE_ID", launchContext.surfaceId)
+		}
+	}
+	if os.Getenv("OPENCODE_EXPERIMENTAL_BACKGROUND_SUBAGENTS") == "" {
+		os.Setenv("OPENCODE_EXPERIMENTAL_BACKGROUND_SUBAGENTS", "true")
+	}
+
+	port := openCodeNativeEffectivePort(args)
+	os.Setenv("OPENCODE_PORT", port)
+	launchArgs := openCodeNativeLaunchArgs(args, port)
+	launchPath, launchArgv := resolveNodeScriptExec(opencodePath, launchArgs, originalPath, "")
+	if err := syscall.Exec(launchPath, launchArgv, os.Environ()); err != nil {
```

golangci-lint (2.12.2)

[medium] G204: Subprocess launched with variable

(gosec)